src/updateTodo.js

```diff
@@ -0,0 +1,29 @@
+export default function handleCheck(list, listOfTodoElement) {
+  listOfTodoElement.forEach((element) => {
+    const checkInputField = element.children[0].children[0];
+    const descriptionTag = element.children[0].children[1];
+
+    checkInputField.addEventListener('change', () => {
+      if (checkInputField.checked) {
+        descriptionTag.classList.add('isChecked');
+        list.forEach((task) => {
+          const matchedIndex = task.index === Number(checkInputField.value);
+          if (matchedIndex) {
+            task.completed = true;
+            checkInputField.checked = true;
+            localStorage.setItem('todos', JSON.stringify(list));
+          }
+        });
+      } else {
+        descriptionTag.classList.remove('isChecked');
+        list.forEach((task) => {
+          const matchedIndex = task.index === Number(checkInputField.value);
+          if (matchedIndex) {
+            task.completed = false;
+            localStorage.setItem('todos', JSON.stringify(list));
```

Comment:
Refactor the if statements into functions that can be reused so you can keep your codebase cleaner.